pyworkflow/gui/plotter.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, x=1, y=1, mainTitle="",
             if mainTitle:
                 self.figure.suptitle(mainTitle, fontsize=fontsize + 4)
             if windowTitle:
-                self.figure.canvas.set_window_title(windowTitle) 
+                self.figure.canvas.manager.set_window_title(windowTitle)
```

Comment:
Do you remember why this change?

Reply:
https://stackoverflow.com/questions/5812960/change-figure-window-title-in-pylab